site/content/contribute/containers/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Containers"
+section: "contribute"
+---
+
+# Mattermost Containers
+
+Mattermost uses the [Docker Registry](https://hub.docker.com/u/mattermost) to publish the official images for the Mattermost Server and also for other supporting images that are used for internal/public development and testing.
+
+This page lists all the Docker repositories currently in use.
+
+## Mattermost Official Docker Images
+
+- [mattermost/mattermost-enterprise-edition](https://hub.docker.com/r/mattermost/mattermost-enterprise-edition) - **Official Mattermost Server** image for the **Enterprise Edition version**. To find the Dockerfile please refer to https://github.com/mattermost/mattermost-server/tree/master/build.
+
+- [mattermost/mattermost-team-edition](https://hub.docker.com/r/mattermost/mattermost-team-edition) - **Official Mattermost Server** image for the **Team Edition version**. To find the Dockerfile please refer to https://github.com/mattermost/mattermost-server/tree/master/build.
+
+- [mattermost/mattermost-push-proxy](https://hub.docker.com/r/mattermost/mattermost-push-proxy) - Mattermost Push Proxy. [Documentation](https://developers.mattermost.com/contribute/mobile/push-notifications/service/). [GitHub](https://github.com/mattermost/mattermost-push-proxy).
+
+- [mattermost/mattermost-loadtest](https://hub.docker.com/r/mattermost/mattermost-loadtest) - Image for the Load Test application. Tools for profiling Mattermost under heavy load. [GitHub](https://github.com/mattermost/mattermost-load-test).
+
+- [mattermost/mattermost-operator](https://hub.docker.com/r/mattermost/mattermost-operator) - Official image for Mattermost Operator for Kubernetes. For more information see https://github.com/mattermost/mattermost-operator.
+
+- [mattermost/mattermost-cloud](https://hub.docker.com/r/mattermost/mattermost-cloud) - Mattermost Private Cloud is a SaaS offering meant to smooth and accelerate the customer journey from trial to full adoption. For more information see https://github.com/mattermost/mattermost-cloud.
+
+## Community-Maintained Docker Images
+
+- [mattermost/mattermost-prod-app](https://hub.docker.com/r/mattermost/mattermost-prod-app) - Community driven image for Mattermost Server. **This Docker repository will be deprecated in Mattermost 6.0**. For more information and to check the Dockerfile please refer to https://github.com/mattermost/mattermost-docker.
```

Comment:
To confirm, is this being tracked by a Jira ticket?

Reply:
Currently tracked in a 6.0 doc https://docs.google.com/document/d/15Q9sNBSy8aPOS57dkFXDCcdL81UxAWuyj1KwbqGhpvc/edit

I believe 6.0 work will be moved over to Jira once plans are finalized